ci/conda_env_python.yml

```diff
@@ -19,9 +19,11 @@ cython=0.29.7
 cloudpickle
 hypothesis
 numpy>=1.14
+minio
```

Comment:
Hmm... is this necessary? I'd say either we download it from upstream, or install a conda package... we don't need two different ways of installing it.

Reply:
`minio` here is the python minio client (more convenient then boto3), whereas the travis script downloads the minio server.

Reply:
Ah, thanks. I didn't realize.